src/sqlfluff/dialects/dialect_mysql.py

```diff
@@ -2333,3 +2360,23 @@ class DropTriggerStatementSegment(ansi.DropTriggerStatementSegment):
         Ref("IfExistsGrammar", optional=True),
         Ref("TriggerReferenceSegment"),
     )
+
+
+class ColumnReferenceSegment(ansi.ColumnReferenceSegment):
```

Comment:
Since this is unusual syntax, could you add a link to the MySQL documentation for this?

Reply:
Thanks. Added in https://github.com/sqlfluff/sqlfluff/pull/3864/commits/d0fc8f08f0abb5a5070facc57cb68e90948ec51d